stm32cube/stm32h5xx/README

```diff
@@ -54,4 +54,12 @@ Patch List:
        drivers/include/stm32h5xx_ll_spi.h
       ST internal bug : 147754
 
+ *fix to the V2 HAL API to get PTP to work
+     In the HAL_ETH_ReadData function where it checks for the last descriptor, 
+     we added a checked if the TSA bit was set in DESC1
+     If the TSA bit is set then have a peak at the context descriptor which should be the one 
+     after the last descriptor
+     If the CTXT bit is set in the context descriptor then extract the timestamps
+     ST internal bug : 161504
```

Comment:
Please add the impacted file before this line

Reply:
ok